LocalSettings.php

```diff
@@ -534,6 +536,15 @@
 				'consul',
 			),
 		),
+		'+cinemawiki' => array(
+			'bureaucrat' => array(
+				'editor',
```

Comment:
@TheTVFan It would be easier if you come on IRC and then SPF can explain it to you.